app/serializers/search/postgres_comment_serializer.rb

```diff
@@ -0,0 +1,41 @@
+module Search
+  # TODO[@atsmith813]: Rename this to CommentSerializer once Elasticsearch is removed
+  class PostgresCommentSerializer < ApplicationSerializer
+    attribute :id, &:search_id
+
+    attributes :path, :public_reactions_count
```

Comment:
```suggestion
    attributes :path do |comment, params|
      user = params[:users][comment.user_id]

      if user
        "/#{user.username}/comment/#{comment.id_code_generated}"
      else
        "/404.html"
      end
    end
    
    attributes :public_reactions_count    
```

Reply:
Will do! We'll have to remember to change this if we ever change the `path` logic on the `Comment` model 😅